fastlane/lib/fastlane/actions/docs/capture_ios_screenshots.md

```diff
@@ -336,3 +336,22 @@ When the app dies directly after the application is launched there might be 2 pr
 ## Determine language
 
 To detect the currently used localization in your tests, access the `deviceLanguage` variable from `SnapshotHelper.swift`.
+
+## Speed up snapshots
+
+A lot of time in UI tests is spent waiting for animations.
+
+You can disable `UIView` animations in your app to make the tests faster:
+
+```swift
+if ProcessInfo().arguments.contains("SKIP_ANIMATIONS") {
```

Comment:
@xavierLowmiller Do we want to mention about putting this value in `launch_arguments`? Otherwise users might assume this is built in by default 🤔

Reply:
Yes, good point.

The launch arguments can be passed
* via ruby: `snapshot(launch_arguments: ["SKIP_ANIMATIONS"])`
* via Swift: `app.launchArguments = ["SKIP_ANIMATIONS")`

Which should I use in the documentation?

Reply:
Yeah yeah, if you want to put the ruby one in the doc that would be 🔥 And then we can merge this!